tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -273,7 +312,28 @@ def LoadTest(config, test_dir=get_root_dir()):
             try:
                 all_ini_sections.remove(section)
                 test_config.add_logical_test(sect.get('Name', ''), tds_name, sect.get(KEY_EXCLUSIONS, ''),
-                                             sect.get('TestPath', ''), test_dir, get_password_file(sect), get_expected_message(sect))
+                                             sect.get('TestPath', ''), test_dir, get_password_file(sect),
+                                             get_expected_message(sect), get_is_smoke_test(sect),
+                                             get_is_test_enabled(sect), False)
+            except KeyError as e:
+                logging.debug(e)
+                pass
+        # Add smoke tests
+        elif connection_test in section:
+            staples_tds_name = 'Staples.*.tds'.replace('*', config_name)
```

Comment:
Since these use the standard tds files you can just pass in CALCS_TDS or STAPLES_TDS and it does the substitution later.

Reply:
Whoops! Refactored in 6a7ed14 below.